### PR TITLE
Feature/claim collectibles

### PR DESCRIPTION
**Pull request by hachiojidev**
Two new NFT's to claim

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -25,7 +25,8 @@
     "@typescript-eslint/explicit-module-boundary-types": 0,
     "@typescript-eslint/no-explicit-any": 0,
     "radix": 0,
-    "import/no-extraneous-dependencies": 0
+    "import/no-extraneous-dependencies": 0,
+    "jsx-a11y/media-has-caption": 0
```
Comment (hachiojidev) [inline .eslintrc:29]: For the `video` tag

File: package.json [hunk 2]
```diff
@@ -41,6 +41,7 @@
     "@types/react-router-dom": "^5.1.5",
     "@types/styled-components": "^5.1.7",
     "bignumber.js": "^9.0.0",
+    "canvas-confetti": "^1.3.3",
```
Comment (hachiojidev) [inline package.json:44]: Given the api on this there is no point in my opinion to put this in the UI Kit.

File: src/config/constants/nfts.ts [hunk 3]
```diff
@@ -1,6 +1,34 @@
...
+      md: 'bullish-md.png',
+      sm: 'bullish-sm.png',
+      ipfs: 'https://gateway.pinata.cloud/ipfs/QmNS1A5HsRW1JvFWtGkm4o9TgZVe2P7kA8TB4yxvS6A7ms/bullish.png',
+    },
+    video: {
+      webm: 'https://gateway.pinata.cloud/ipfs/QmNS1A5HsRW1JvFWtGkm4o9TgZVe2P7kA8TB4yxvS6A7ms/bullish.webm',
+      mp4: 'https://gateway.pinata.cloud/ipfs/QmNS1A5HsRW1JvFWtGkm4o9TgZVe2P7kA8TB4yxvS6A7ms/bullish.mp4',
+    },
+    sortOrder: 999,
+    bunnyId: 11,
```
Comment (hachiojidev) [inline src/config/constants/nfts.ts:18]: @Chef-Chungus This is correct right? Don't want another mistake :)

File: src/config/constants/nfts.ts [hunk 4]
```diff
@@ -1,6 +1,34 @@
...
+    name: 'Hiccup',
+    description: "Oopsie daisy! Hiccup's had a bit of an accident. Poor little fella.",
+    images: {
+      lg: 'hiccup-lg.png',
+      md: 'hiccup-md.png',
+      sm: 'hiccup-sm.png',
+      ipfs: 'https://gateway.pinata.cloud/ipfs/QmNS1A5HsRW1JvFWtGkm4o9TgZVe2P7kA8TB4yxvS6A7ms/hiccup.png',
+    },
+    sortOrder: 999,
+    bunnyId: 10,
```
Comment (hachiojidev) [inline src/config/constants/nfts.ts:30]: @Chef-Chungus Same here.

File: package.json (hunk 2 above)
Comment (RabbitDoge) [reply package.json:44]: agree

**Review by RabbitDoge — APPROVED** [review]

**Pull request merged**